server/src/main/scala/za/co/absa/atum/server/api/database/DoobieImplicits.scala

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.api.database
+
+import cats.Show
+import cats.data.NonEmptyList
+
+import doobie.{Get, Put}
+import doobie.postgres.implicits._
+
+import org.postgresql.jdbc.PgArray
+import org.postgresql.util.PGobject
+
+import scala.util.{Failure, Success, Try}
+
+object DoobieImplicits {
```

Comment:
todo I just want to ask about some things related to this module

Reply:
Discussed on the call, I think we can resolve it.